src/main.zig

```diff
@@ -221,7 +224,33 @@ pub fn main() anyerror!void {
         const empty = [_][:0]u8{};
         break :file_result &empty;
     };
+    
+    //do the Operation
+    try doOperation(
+        allocator, 
+        operation, 
+        archive_path, 
+        archive_type, 
+        stderr, stdout, 
+        modifiers, 
+        files
+    );
+
+    
+}
 
+pub fn doOperation(
```

Comment:
```suggestion
pub fn performOperation(
```

I think performOperation would be more descriptive.